### PR TITLE
Add release workflow

### PR DESCRIPTION
**Pull request by DenDen039**
##### SUMMARY
Adds release workflow. Creates Github Release and uploads collection to Ansible Galaxy and Automation Hub.
##### ISSUE TYPE
- Feature Pull Request

##### COMPONENT NAME
.github/workflows

### CODE REVIEW

File: .github/workflows/create-release.yaml [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+    if: github.event.pull_request.merged == true && contains(github.event.pull_request.labels.*.name, 'release')
+    runs-on: ubuntu-latest
+    environment: fusion-env
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: Get version from galaxy.yml
+        run: |
+          RELASE_VERSION=$(grep 'version:' galaxy.yml | awk '{print $2}')
```
Comment (sdodsley) [inline .github/workflows/create-release.yaml:19]: nit: RELEASE

File: .github/workflows/create-release.yaml [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+          python -m pip install --upgrade pip
+          pip install ansible ansible-lint yamllint ansible-galaxy ansible-galaxy-importer
+
+      - name: Build Ansible Collection
+        run: ansible-galaxy collection build
+
+      - name: Create a release
+        env:
+          GITHUB_TOKEN: ${{ github.TOKEN }}
+        run: gh release create "${{ env.RELEASE_VERSION }}" --title "Fusion Collection v${{ env.RELEASE_VERSION }}" --notes-file changelog
```
Comment (sdodsley) [inline .github/workflows/create-release.yaml:67]: So we only need the changelog for this release version., not the whole changelog

Comment (DenDen039) [reply .github/workflows/create-release.yaml:67]: I have a step for extracting change log 
https://github.com/DenDen039/Fusion-Collection/blob/0f570add89294731045d7e165ca9a091477c7ff5/.github/workflows/create-release.yaml#L34

Comment (sdodsley) [reply .github/workflows/create-release.yaml:62]: Cool

File: .github/workflows/create-release.yaml [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+          url=${{ secrets.RH_AUTOMATION_HUB_URL }}
+          auth_url=${{ secrets.RH_SSO_URL }}
+          token=${{ secrets.RH_AUTOMATION_HUB_TOKEN }}
+          EOF
+
+      - name: Release to Automation Hub
+        run: ansible-galaxy collection publish ./purestorage-fusion-${{ env.RELEASE_VERSION }}.tar.gz
+
+  create_execution_release:
+    needs: [publish_collection_to_galaxy, publish_collection_to_automation_hub]
```
Comment (sdodsley) [inline .github/workflows/create-release.yaml:110]: technically this only requires the collection to be in galaxy

**Review by sdodsley — DISMISSED** [review]

**Pull request merged**